src/engines/html5/html5.js

```diff
@@ -941,4 +947,57 @@ export default class Html5 extends FakeEventTarget implements IEngine {
     }
     return extended;
   }
+
+  enterPictureInPicture(): void {
+    try {
+      // Currently it's supported in chrome and in safari. So if we consider checking support before,
+      // we can use this flag to distinguish between two. In the future we might need a different method.
+      if (document.pictureInPictureEnabled) {
+        this._el
+          .requestPictureInPicture()
```

Comment:
use this promise and add `then` to set `_isInPictureInPicture` to true and false in `catch` if it throws